cranelift/codegen/meta/src/gen_legalizer.rs

```diff
@@ -711,16 +712,22 @@ pub(crate) fn generate(
         fmt.update_file(format!("{}-{}.rs", filename_prefix, isa.name), out_dir)?;
     }
 
+    // Add extra legalization groups that were explicitly requested.
+    for group in extra_legalization_groups {
+        shared_group_names.insert(group);
+    }
+
     // Generate shared legalize groups.
     let mut fmt = Formatter::new();
+    // Generate shared legalize groups.
     let mut type_sets = UniqueTable::new();
     let mut sorted_shared_group_names = Vec::from_iter(shared_group_names);
     sorted_shared_group_names.sort();
     for group_name in &sorted_shared_group_names {
         let group = transform_groups.by_name(group_name);
-        gen_transform_group(group, transform_groups, &mut type_sets, &mut fmt);
+        gen_transform_group(group, transform_groups, &mut type_sets, fmt);
```

Comment:
Looks like we still need `&mut fmt`: https://github.com/bytecodealliance/wasmtime/pull/2003/checks?check_run_id=858163032#step:11:115